part1/chapter6/quiz1/questions/question_2.rb

```diff
@@ -20,3 +20,35 @@
 # * Note: You can assume the players will only ever input `rock`,
 #   `paper` or `scissors`.
 
+paper = "paper"
+rock = "rock"
+scissors = "scissors"
+
+while true do
+  puts "Player 1 choose between rock, paper and scissors:"
+  player_1 = gets.chomp.downcase
```

Comment:
There is probably a better variable name than `player_1` since the thing it’s storing isn’t a player. (Instil good naming habits early - they will save you much confusion when you're debugging!)